src/extensions/manifest.ts

```diff
@@ -92,3 +95,55 @@ async function writeEnvFiles(
     await config.askWriteProjectFile(`extensions/${spec.instanceId}.env`, content, force);
   }
 }
+
+/**
+ * readParams gets the params for an extension instance from the `extensions` folder,
+ * checking for project specific env files, then falling back to generic env files.
+ * This checks the following locations & if a param is defined in multiple places, it prefers
+ * whichever is higher on this list:
+ *  - extensions/{instanceId}.env.{projectID}
+ *  - extensions/{instanceId}.env.{projectNumber}
+ *  - extensions/{instanceId}.env.{projectAlias}
+ *  - extensions/{instanceId}.env
```

Comment:
Consider adding .local to this list with a note it only will be checked in checkLocal is true

Reply:
sg, added to top of the list